moped-editor/src/views/projects/projectView/ProjectNameForm.js

```diff
@@ -0,0 +1,211 @@
+    setProjectName(e.target.value);
+    if (!e.target.value) {
+      setPrimaryTitleError(true);
+    } else {
+      setPrimaryTitleError(false);
+    }
+  };
+
+  const handleSecondaryNameChange = (e) => {
+    // Update the secondary name state variable when the input changes
+    setSecondaryName(e.target.value);
+  };
+
+  return (
+    <Grid container spacing={4}>
+      {/* Primary project name field */}
+      <Grid item xs={12} sm={6}>
+        <form onSubmit={(e) => handleAcceptClick(e)}>
+          <TextField
+            required
+            autoFocus
+            variant="standard"
+            fullWidth
+            id="project_name"
+            label={"Project name"}
+            type="text"
+            value={projectName}
+            error={primaryTitleError}
+            placeholder={
+              primaryTitleError ? "Title cannot be blank" : "Enter project name"
+            }
+            multiline={false}
+            rows={1}
+            onChange={handleProjectNameChange}
+            InputLabelProps={{
+              shrink: true,
+            }}
+            InputProps={{
+              classes: {
+                input: classes.projectNameEditField,
+              },
+            }}
+          />
+        </form>
+      </Grid>
+
+      {/* Secondary project name field */}
+      <Grid item xs={12} sm={3}>
+        <form onSubmit={(e) => handleAcceptClick(e)}>
+          <TextField
+            variant="standard"
+            fullWidth
+            id="secondary_name"
+            label={"Secondary name"}
+            type="text"
+            value={secondaryName}
+            placeholder={"Secondary name"}
+            multiline={false}
+            rows={1}
+            onChange={handleSecondaryNameChange}
+            InputLabelProps={{
+              shrink: true,
+            }}
+            InputProps={{
+              classes: {
+                input: classes.projectSecondaryNameEditField,
+              },
+            }}
+          />
+        </form>
+      </Grid>
+
+      {/* Accept / Cancel icons.
+      This grid item gets a minimum width to prevent it from reflowing onto two lines. */}
+      <Grid
+        item
+        container
+        direction="row"
+        justifyContent="flex-end"
+        alignItems="flex-end"
+        xs={12}
+        sm={1}
+        sx={{ minWidth: "77px" }}
+      >
+        <Icon
+          className={classes.editIcons}
+          onClick={(e) => handleAcceptClick(e)}
+        >
+          check
+        </Icon>
+        <Icon
+          className={classes.editIcons}
+          onClick={(e) => handleCancelClick(e)}
+        >
+          close
+        </Icon>
+      </Grid>
+
+      {/* The status badge. Here, we're going to jog it down a bit to make it visually centered
+      along the horizontal midline of the project name input field. */}
+      <Grid item xs={12} md={2}>
+        <Box sx={{ display: "inline", position: "relative", top: "1rem" }}>
```

Comment:
Since the parent `<Grid item>` here uses flexbox, you could probably get away removing this Box and using `alignContent: end` on the grid item if the intention is to push the status badge down so it aligns with the bottom of the inputs.

Reply:
I appreciate your feedback on using MUI better; I'm so green at it. 💯 https://github.com/cityofaustin/atd-moped/pull/1302/commits/400416a464148c546dec209e3946881de930efb1